octoprint_octorelay/static/js/octorelay.js

```diff
@@ -1,37 +1,33 @@
-$(function() {
-    function OctoRelayViewModel(parameters) {
+"use strict";
+$(function () {
+    var OctoRelayViewModel = function (_a) {
+        var settingsViewModel = _a[0], loginStateViewModel = _a[1];
         var self = this;
-        self.settingsViewModel = parameters[0]
-        self.loginState = parameters[1];
-
-        self.onDataUpdaterPluginMessage = function(plugin, data) {
+        self.settingsViewModel = settingsViewModel;
+        self.loginState = loginStateViewModel;
+        self.onDataUpdaterPluginMessage = function (plugin, data) {
             if (plugin !== "octorelay") {
                 return;
             }
-            // console.log("onDataUpdaterPluginMessage: " + JSON.stringify(data));
-            for (const [key, value] of Object.entries(data)) {
-                // console.log(JSON.stringify(key));
-                // console.log(JSON.stringify(value));
+            for (var _i = 0, _a = Object.entries(data); _i < _a.length; _i++) {
+                var _b = _a[_i], key = _b[0], value = _b[1];
+                var btn = $("#relais" + key);
                 if (value.active !== undefined) {
-                    if (value.active === 1) {
-                        $("#relais" + key).show();
-                    } else {
-                        $("#relais" + key).hide();
-                    }
+                    btn.toggle(value.active === 1);
                 }
+                var icon = $("#ralayIcon" + key);
                 if (value.iconText !== undefined) {
-                    $("#ralayIcon" + key).html(value.iconText);
+                    icon.html(value.iconText);
                 }
                 if (value.labelText !== undefined) {
-                    $("#ralayIcon" + key).attr("title", value.labelText);
+                    icon.attr("title", value.labelText);
                 }
                 if (value.confirmOff !== undefined) {
-                    $("#ralayIcon" + key).attr("data-confirm", value.confirmOff);
+                    icon.attr("data-confirm", value.confirmOff);
```

Comment:
Extracting `$("#ralayIcon" + key)` into a const in order to query document once for that element.